cellarium/cas/client.py

```diff
@@ -417,8 +419,13 @@ def annotate_anndata(
             `Default:` ``None``
         :param include_dev_metadata: Boolean indicating whether to include a breakdown of the number of cells
             by dataset
+
         :return: A list of dictionaries with annotations for each of the cells from input adata
         """
+        warnings.warn(
```

Comment:
...re my previous comment, it would take care of this kind of thing for us I think